<launch>
<node name="rc_controller" pkg="rc_control" type="rc_control" output="screen">
      <param name="start_offboard_pub_topic" value="/uav1/mavros/rc/in"/>
      <param name="target_object_position_pub_topic" value = "/ref_uav/pose"/>
      <param name="start_offboard_ch" value="5"/>
      <param name="land_ch" value="6"/>
      <param name="rc_start_mission_ch" value ="7"/>
      <param name="rc_change_mission_ch" value ="8"/>
</node>
</launch>
